src/user-actions.ts

```diff
@@ -0,0 +1,45 @@
+export default interface User {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+  isDeleted: boolean;
+}
+
+interface UserRes {
```

Comment:
I would avoid word abbreviations. What made you use `UserRes` instead of `UserResponse`?

Reply:
fixed